app/utils/discogs.schemas.ts

```diff
@@ -1,8 +1,10 @@
 import z from "zod";
 
 const AlbumSchema =  z.object({
+  cover_image:z.string(),
+  title: z.string().optional(),
   year: z.string().optional(),
-  id: z.number().optional()
+  id: z.number().optional(),
```

Comment:
```suggestion
  id: z.number()
```

this should actually always be defined since it's a unique identifier for the record. If they aren't always defined from the discogs API, I would generate one

Reply:
Tested and there don't seem to be cases where there is no Id